docs/deploy.md

```diff
@@ -207,3 +170,77 @@ for item in response:
                                                                       item.score))
 
 ```
+
+## Presidio As a Service with Kubernetes
+
+### Requirements
+
+1. A Kubernetes 1.9+ cluster with [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) enabled.
+
+   - Note the pod's resources requirements (CPU and memory) and plan the cluster accordingly.
+
+2. [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) installed
+
+   - verify you can communicate with the cluster by running:
+
+     ```sh
+     kubectl version
+     ```
+
+3. Local [helm](https://helm.sh/) client.
+4. Optional - Container Registry - such as [ACR](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-intro).
+5. Recent presidio repo is cloned on your local machine.
+
+### Single click deployment
+
+1. Navigate into `<root>\deployment` from command line.
+
+2. If You have helm installed, but havn't run `helm init`, execute [deploy-helm.sh](../deployment/deploy-helm.sh) in the command line. It will install `tiller` (helm server side) on your cluster, and grant it sufficient permissions.
+
+```sh
+deploy-helm.sh
+```
+
+3. Optional - Grant the Kubernetes cluster access to the container registry. This step can be skipped and the script will pull the container images from [Microsoft syndicates container catalog](https://azure.microsoft.com/en-in/blog/microsoft-syndicates-container-catalog/)
```

Comment:
Maybe note that this is optional only if you plan to use the default images (and not create ones yourself)

Reply:
done